docs/procedures.md

```diff
@@ -61,34 +61,38 @@ Execute the following commands on your local machine, not production.
         $ git checkout main
         $ git pull origin main
 
-4.  Merge `develop` into `main` branch (be careful, as there are sometimes conflicts that need to be manually resolved):
+4. Create a release branch `release/vX.Y.Z` from `develop`. Major/minor release branches should be based on the `HEAD` of `develop`, but bugfix releases may be based on older commits to avoid including features intended for the next major/minor release.
```

Comment:
> on older commits...

"... such as previous release branch.

For more details on managing branches, see https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow"

Some ideas on extending this sentence.